tinc.yaml

```diff
@@ -0,0 +1,3 @@
+dependencies:
+  - name: riak-protobuf
+    path: protobuf
```

Comment:
@llelf I just had quick look at tinc, it's certainly in the same vein as mafia.

Mafia has the same guarantee that packages with the same transitive dependencies are not installed twice. Packages are stored in `~/.ambiata/mafia/packages/<cache-layout-version>/<ghc-version>/<package-id>-<dependency-hash>` and then symlinked in to the local projects sandbox as required.

The main difference I can see at a glance is that tinc doesn't support caching of source dependencies? I could be wrong though as I just had a quick look. This is really useful for us at Ambiata as we have around 150 internal Haskell packages which, all of which reference one and another as git submodules / source dependencies.

I noticed that tinc supports freeze files which is kind of neat, something we've been talking about adding to mafia. Another difference is that we don't use a yaml file or anything for configuration, it's just by convention, so all git submodules with .cabal files in them are assumed to be fair game for source dependencies. 

The docs are pretty bad for mafia at the moment, but if you want to give it a try I'd be happy to assist, just drop me a github issue on the [mafia repo](http://github.com/ambiata/mafia) if you get stuck :)